packages/react-devtools-shared/src/hook.js

```diff
@@ -473,6 +470,33 @@ export function installHook(target: any): DevToolsHook | null {
     }
   }
 
+  function setStrictMode(rendererID, isStrictMode) {
+    const rendererInterface = rendererInterfaces.get(rendererID);
+    if (rendererInterface != null) {
+      if (isStrictMode) {
+        rendererInterface.patchConsoleForStrictMode();
+      } else {
+        rendererInterface.unpatchConsoleForStrictMode();
+      }
+    } else {
```

Comment:
So this `else` block is when a Renderer has _injected_ itself, but the DevTools backend hasn't finished initializing and so it hasn't _attached_ a DevTools interface yet?

Reply:
Yup